docs/static/spec/openapi/logstash-api.yaml

```
@@ -32,52 +32,195 @@ security:
 tags:
   - name: health
     x-displayName: Health report
-    # description: 
+    # description:
     # externalDocs:
-    #   description: 
-    #   url: 
+    #   description:
+    #   url:
   - name: hot threads
     x-displayName: Hot threads
-    # description: 
+    # description:
     # externalDocs:
-    #   description: 
-    #   url: 
+    #   description:
+    #   url:
   - name: node info
     x-displayName: Node info
     description: |
       Logstash node info, like pipeline settings, OS info, and JVM info.
     # externalDocs:
-    #   description: 
-    #   url: 
+    #   description:
+    #   url:
   - name: node stats
     x-displayName: Node stats
     description: |
       Node stats, like JVM stats, process stats, event-related stats, and pipeline runtime stats.
     # externalDocs:
-    #   description: 
-    #   url: 
+    #   description:
+    #   url:
   - name: plugin info
     x-displayName: Plugins info
     description: |
       Plugin info, including a list of installed plugins.
     # externalDocs:
-    #   description: 
-    #   url: 
-paths: 
-  /_node/<types>:
+    #   description:
+    #   url:
+paths:
+  /_node/jvm:
     get:
-      summary: Get node info
+      summary: Gets node-level JVM info, including info about threads.
       description: |
-        Get information about Logstash nodes, where `<types>` (optional) specifies the types of node info you want returned.
-        You can limit the info that is returned by combining any of these types in a comma-separated list:
+        Show node-level JVM stats, such as the JVM process id, version, VM info, memory usage, and info about garbage collectors
```

We do use periods in the descriptions so i've added it here. It might be overkill but I've spelled out the first usage of JVM for clarity too. Should it be changed to "...memory usage, threads, and info..." too?
```suggestion
        Show node-level statistics for the Java virtual machine (JVM), such as the JVM process id, version, VM info, memory usage, and info about garbage collectors.
```

---

it is overkill. JVM is a better word here I believe

---

added the period. thanks